standards/examples/turbogeth.md

```diff
@@ -0,0 +1,72 @@
+## Turbogeth managed service
+
+This article contains examples of Demands & Offers which cover the a Managed Service example (turbogeth).
+
+Compared to Standards v0.6.0, the sample includes a couple of extensions:
+
+### Perpetual Agreement
+
+No `golem.srv.comp.expiration` property specified on the Demand side indicates no expected Agreement end date.
+
+### "Eth" Application Namespace
+
+Any "Ethereum-specific" properties would be maintained in this namespace. This sample includes:
+
+`golem.srv.app.eth.rpc-port` - Indication of RPC endpoint port exposed by the provisioned Geth service.
+
+### Managed Turbogeth runtime
+
+`golem.runtime.name=turbogeth-managed` - Indicated a Managed Turbogeth service to be launched
```

Comment:
just out of curiosity -> isn't this the name of the exeunit runtime to run? wouldn't that mean that it should refer to the wrapper exeunit? 

otherwise, how will we assure uniqueness of the runtime names? plus, should we version the runtimes somehow?

Reply:
Yes, that is the indication of the service wrapper exeunit. In reality we see this as yet another "runtime", controlled bu the universal "supervisor" module.

We do version runtimes (`golem.runtime.version` prop...)